src/App.tsx

```diff
@@ -1,16 +1,26 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd:(movie:Movie)=> void = (movie) => {
```

Comment:
use more descriptive name like `addMovie`

something strange with this typing, let's simplify if.
```suggestion
  const addMovie:(movie: Movie) => {
```

Reply:
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/130560419/08a9eb29-5f9f-4a41-a1fb-594ed6249c84)
for tests, this function should be called onAdd, if I understand correctly